cartridge-cli.lua

```diff
@@ -451,7 +456,10 @@ local function detect_git_version(source_dir)
         return nil
     end
 
-    local raw_version, err = check_output('cd "%s" && git describe --tags --long', source_dir)
+    local raw_version, err = check_output(
```

Comment:
check_output/call/... should call escape internally. You can't escape every argument by hands. In future it's easy to miss it.

Reply:
In future we will use Go.